Format issues

jagdish-15 opened the pull request:
While running `format.mjs` for the other PRs that update the tests, I noticed it was also modifying unrelated files. I believe this PR will address that issue.  

Do you know what might have caused it?  
@SleeplessByte @Cool-Katt  

---

Edit:
Now this check is failing here.  
The command that caused these changes was:  

```bash
corepack pnpm dlx cross-env node scripts/format.mjs
```

SleeplessByte commented:
Does that also show when you run `corepack pnpm dlx cross-env node scripts/format.mjs`?

jagdish-15 commented:
> Does that also show when you run `corepack pnpm dlx cross-env node scripts/format.mjs`?

Yes, it changes the following files when run:

```
$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   concepts/callbacks/about.md
        modified:   concepts/closures/about.md
        modified:   exercises/concept/amusement-park/.meta/design.md
        modified:   exercises/concept/bird-watcher/.meta/design.md
        modified:   exercises/concept/high-score-board/.meta/design.md
        modified:   exercises/concept/mixed-juices/.meta/design.md
        modified:   exercises/concept/ozans-playlist/.meta/design.md

no changes added to commit (use "git add" and/or "git commit -a")
```

And these files, upon running the `/format` here, go back to normal!

SleeplessByte commented:
(Maybe related: https://github.com/exercism/javascript/pull/2740)
